### ISSUE TITLE
Image Preprocessing to finetune/predict using pre trained VGG model

### ISSUE THREAD

**Issue by sudhashbahu** [tflearn/tflearn#391]
The example at https://github.com/tflearn/tflearn/blob/master/examples/images/vgg_network_finetuning.py shows how to fine tune the pre-trained VGG16 model.

However the trained model might have applied mean subtraction and Std normalization where the mean/STD values are calculated over entire training dataset.

if we need to fine-tune the model (or use it as-is for classification) shouldn't we have to apply the exact same preprocessing to new images?

The fine tuning example above applied Normalization to the input image but this will not match that over the entire pre-training dataset.

**Comment by aymericdamien** [comment]
You are right! this was missing. I updated the example, you can have a look.

**Comment by sudhashbahu** [comment]
Thank you for the quick action.

**Comment by sudhashbahu** [comment]
Are you sure the Normalize=True should be retained in the image preloader?

**Comment by aymericdamien** [comment]
Ops, that is a typo, thanks for pointing that out! I fixed it.
